src/components/common/Field.tsx

```diff
@@ -131,3 +133,56 @@ export const SelectField: React.FC<SelectFieldProps> = ({
     </div>
   );
 };
+
+type SwitchFieldProps = {
+  fieldClassName?: string;
+  switchLabel?: string;
+  tip?: string;
+  error?: string;
+} & SwitchProps &
+  HTMLProps<HTMLInputElement>;
+
+export const SwitchField: React.FC<SwitchFieldProps> = ({
```

Comment:
@yuchenshi this currently renders as black when it's in the "on" position:

<img width="116" alt="Screen Shot 2022-01-19 at 1 33 17 PM" src="https://user-images.githubusercontent.com/3759507/150192655-4a019bd1-77ef-43cc-be16-b325f994f8dd.png">

How do I make it purple? Black doesn't seem to be the default for RMWC, so it seems to be picking up a theme from somewhere, but I'm not sure where.

Reply:
I'd need hands-on access to debug this one. Let me put this in the known issues for now and we can peer-program later when we get to minor UX fixes. For now, consider it non-blocking.